project_status/views/project.xml

```diff
@@ -0,0 +1,93 @@
+<odoo>
+  <!-- List View-->
+  <record id="project_status_view_inherit_list" model="ir.ui.view">
+    <field name="name">project.status.tree</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.view_project"/>
+    <field name="type">tree</field>
+    <field name="field_parent">child_ids</field>
+    <field name="arch" type="xml">
+      <field name="user_id" position="after">
+        <field name="project_status"/>
+      </field>
+    </field>
+  </record>
+
+  <!-- Form View -->
+  <record id="project_status_view_inherit_form" model="ir.ui.view">
+    <field name="name">project.status.form</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.edit_project"/>
+    <field name="type">form</field>
+    <field name="arch" type="xml">
+      <data>
+        <xpath expr="//header" position="inside">
+          <field name="project_status"
+                 widget="statusbar"
+                 options="{'clickable': '1', 'fold_field': 'fold'}"/>
+        </xpath>
+      </data>
+    </field>
+  </record>
+
+  <!-- Simplified Form View -->
+  <record id="project_status_view_inherit_simple" model="ir.ui.view">
+    <field name="name">project.status.simpleform</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.project_project_view_form_simplified"/>
+    <field name="type">form</field>
+    <field name="arch" type="xml">
+      <data>
+        <field name="name" position="after">
+          <field name="project_status"
+                 widget="selection"
+                 required="1"/>
+        </field>
+      </data>
+    </field>
+  </record>
+
+  <!-- Kanban View -->
+  <record id="project_status_view_inherit_kanban" model="ir.ui.view">
+    <field name="name">project.status.kanban</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.view_project_kanban"/>
+    <field name="arch" type="xml">
+      <data>
+        <xpath expr="//div[hasclass('o_primary')]" position="after">
+          <div>
+            <t t-if="record.project_status.raw_value">
+              <span class="o_label"> (</span>
+              <span class="o_value"><field name="project_status"/></span>
+              <span class="o_label">)</span>
+            </t>
+          </div>
+        </xpath>
+      </data>
+    </field>
+  </record>
+
+  <!-- Search View -->
+  <record id="project_status_view_inherit_search" model="ir.ui.view">

```

Comment:
Not required for approvement, but would be nice if you use the original id

Reply:
I've changed the id to the original's id. Thanks.